src/abstractdataframe/io.jl

```diff
@@ -82,7 +82,8 @@ function _show(io::IO, ::MIME"text/html", df::AbstractDataFrame;
         if size(df, 2) == 0
             rowid = nothing
         else
-            size(df, 1) == 1 || throw(ArgumentError("rowid may be passed only with a single row data frame"))
+            size(df, 1) == 1 || 
```

Comment:
I personally would prefer to use `if` in situations when we do a line break after `||` as it might be confusing when reading the code. Or maybe just split the error message like in other places?

Reply:
Due to an indentation in the next line, I cannot see any problem in reading the broken lines. Is your preferred style a widely accepted practice in Julia community? If so, I have no problem with the change.

Reply:
There isn't a generally accepted style here in the Julia community (that I'm aware of), so it's more up to discussion and package maintainer preference. In this case, I personally don't have a strong preference, so I'm fine with whatever others want to go with.

Reply:
I have just checked in Base that this style is used there, so I think we can leave it as is.